rca/project_styleguide/templates/patterns/organisms/staff-overview/staff-overview--single.html

```diff
@@ -1,8 +1,10 @@
 <div class="section__row grid staff-overview staff-overview--single">
     <h2 class="heading heading--four staff-overview__heading">Staff</h2>
-    {% include "patterns/molecules/staff-detail/staff-detail.html" %}
-    <a class="staff-overview__view-all link link--primary link--link body body--two" href="#">
-        <span class="link__label">See all programme staff</span>
+    {% for item in staff %}
```

Comment:
We probably should pass a queryset from Python that has `select_related('image')`.

Reply:
Thanks @tm-kn, I've added this [here](https://github.com/torchbox/rca-wagtail-2019/pull/96/files#diff-7e356f728d6d9ada8c4d7be20f54d5fcR573) if you could confirm it